server/src/main/java/org/elasticsearch/index/mapper/TypeParsers.java

```diff
@@ -79,49 +75,11 @@ private static void parseAnalyzersAndTermVectors(TextFieldMapper.Builder builder
                 }
                 indexAnalyzer = analyzer;
                 iterator.remove();
-            } else if (propName.equals("search_analyzer")) {
-                NamedAnalyzer analyzer = parserContext.getIndexAnalyzers().get(propNode.toString());
-                if (analyzer == null) {
-                    throw new MapperParsingException("analyzer [" + propNode.toString() + "] not found for field [" + name + "]");
-                }
-                searchAnalyzer = analyzer;
-                iterator.remove();
```

Comment:
Are we safe to delete the `iterator.remove()` call here?  It depends if crate would ever have written a search-time analyzer parameter into the mappings.